data/language/en-GB.txt

```diff
@@ -4449,6 +4449,8 @@ STR_6137    :OpenRCT2, a free and open source clone of Roller Coaster Tycoon 2.
 STR_6138    :OpenRCT2 is the work of many authors, a full list can be found in {OPENQUOTES}contributors.md{ENDQUOTES}. For more information, visit http://github.com/OpenRCT2/OpenRCT2
 STR_6139    :All product and company names belong to their respective holders. Use of them does not imply any affiliation with or endorsement by them.
 STR_6140    :Changelog...
+STR_6141    :RCT1 Bottom Toolbar
+STR_6142    :OpenRCT2
```

Comment:
Just use another string that already reads 'OpenRCT2'. There is really no need for an extra one.